src/services/completions.ts

```diff
@@ -1737,7 +1737,7 @@ function getEntryForObjectLiteralMethodCompletion(
         module: options.module,
         target: options.target,
         omitTrailingSemicolon: false,
-        newLine: getNewLineKind(getNewLineCharacter(options, maybeBind(host, host.getNewLine))),
+        newLine: getNewLineKind(getNewLineCharacter(options)),

```

Comment:
shouldnt this be getNewLineOrDefaultFromHost

Reply:
Yep